tests/phpunit/CRM/Core/I18n/SchemaTest.php

```diff
@@ -112,7 +112,7 @@ public function testSchemaBuild(): void {
     $inUseCollation = CRM_Core_BAO_SchemaHandler::getInUseCollation();
     $testCreateTable = CRM_Core_DAO::executeQuery('SHOW create table civicrm_price_set', [], TRUE, NULL, FALSE, FALSE);
     while ($testCreateTable->fetch()) {
-      $this->assertStringContainsString("`title_en_US` varchar(255) COLLATE {$inUseCollation} NOT NULL DEFAULT '' COMMENT 'Displayed title for the Price Set.'", $testCreateTable->Create_Table);
+      $this->assertStringContainsString("`title_en_US` varchar(255) COLLATE {$inUseCollation} NOT NULL", $testCreateTable->Create_Table);
```

Comment:
I'm still not sure if https://github.com/civicrm/civicrm-core/pull/27226 was the right approach, but assuming it is, then the `DEFAULT ''` is important.

Reply:
Thanks @demeritcowboy I've pushed a 2nd commit to restore that funkyness